index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+  <title>capston</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <section class="first">
+  <header>
+    <div class="flists">
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-facebook-f"></i>
+      <ul class="flist">
+        <li><a href="">English</a></li>
+        <li><a href="">My page</a></li>
+        <li><a href="">logout</a></li>
+      </ul>
+    </div>
+    <div class="bar" id="bar">
+      <i class="fa-solid fa-bars fa-2x"></i>
+    </div>
+    <div class="hide" id="close">
+      <i class="fa-solid fa-xmark "></i>
+    </div>
+    <nav class="lnav">
+      <div class="logo">
+        <h1><a href="#hello">AYC</a></h1>
+      </div>
+      <div>
+          <ul class="barlist">
+            <li><a href="about.html" class="nav-link-1">About</a></li>
+            <li><a href="" class="nav-link-2">Program</a></li>
+            <li><a href="" class="nav-link-3">Join</a></li>
+            <li><a href="" class="nav-link-4">Sponsor</a></li>
+            <li><a href="" class="nav-link-5">News</a></li>
+            <li><button>AYC company</button></li>
+          </ul>
+      </div>
+    </nav>  
+  
+  </header>
+  <div class="hello" id="hello">
+    <h1>Hello!African Youth</h1>
+    <h2>African Youth Confrance 2022</h2>
```

Comment:
- [ ] Please make the `African Youth Conference 2022` text the main `<h1>` tag and `Hello!African Youth` the `<h2>` tag